jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MojoCommon.java

```diff
@@ -140,12 +140,14 @@ static void finishUpdateChecker(
     // Order is important, so use a LinkedHashMap
     Map<String, FilePermissions> permissionsMap = new LinkedHashMap<>();
     for (PermissionConfiguration permission : permissionList) {
-      if (!permission.getFile().isPresent() || !permission.getMode().isPresent()) {
+      Optional<String> permissionFile = permission.getFile();
+      Optional<String> permissionMode = permission.getMode();
```

Comment:
Can be brief in this short local context.
```suggestion
      Optional<String> file = permission.getFile();
      Optional<String> mode = permission.getMode();
```